File: src/main/java/no/nav/familie/ks/oppslag/dokarkiv/metadata/KontanstøtteSøknadVedleggMetadata.java
package no.nav.familie.ks.oppslag.dokarkiv.metadata;


public class KontanstøtteSøknadVedleggMetadata extends AbstractDokumentMetadata {
    public static final String TEMA = "KON";
    public static final String BEHANDLINGSTEMA = null; // https://confluence.adeo.no/display/BOA/Behandlingstema
    public static final String DOKUMENT_TYPE_ID = "KONTANTSTØTTE_SØKNAD_VEDLEGG";
    public static final String DOKUMENT_KATEGORI = "IS";

    @Override
    public String getTema() {
        return TEMA;
    }

    @Override
    public String getBehandlingstema() {
        return BEHANDLINGSTEMA;
    }

    @Override
    public String getKanal() {
        return null;
    }

    @Override
    public String getDokumentTypeId() {
        return DOKUMENT_TYPE_ID;
    }

    @Override
    public String getTittel() {
        return null;
    }

    @Override
    public String getBrevkode() {
        return null;
    }

    @Override
    public String getDokumentKategori() {
        return DOKUMENT_KATEGORI;
    }
}

File: src/main/java/no/nav/familie/integrasjoner/dokarkiv/metadata/KontanstøtteSøknadVedleggMetadata.kt
package no.nav.familie.integrasjoner.dokarkiv.metadata

import no.nav.familie.integrasjoner.dokarkiv.client.domene.JournalpostType
import org.springframework.stereotype.Component

@Component
object KontanstøtteSøknadVedleggMetadata : DokumentMetadata {

    override val journalpostType: JournalpostType = JournalpostType.INNGAAENDE
    override val fagsakSystem: String? = null
    override val tema: String = "KON"
    override val behandlingstema: String? = null // https://confluence.adeo.no/display/BOA/Behandlingstema
    override val kanal: String? = null
    override val dokumentTypeId: String = "KONTANTSTØTTE_SØKNAD_VEDLEGG"
    override val tittel: String? = null
    override val brevkode: String? = null
    override val dokumentKategori: String = "IS"

}